packages/core/src/connectivity/scp-cf/xsuaa-service.ts

```diff
@@ -268,28 +253,40 @@ const tokenKeyKeyMapping: { [key: string]: keyof TokenKey } = {
   n: 'publicKeyModulus'
 };
 
+function headers(authHeader: string): Pick<AxiosRequestConfig, 'headers'> {
+  return {
+    headers: {
+      Authorization: authHeader,
+      'Content-Type': 'application/x-www-form-urlencoded',
+      Accept: 'application/json'
+    }
+  };
+}
+
 function post(
   tokenServiceUrlOrXsuaaServiceCredentials: string | XsuaaServiceCredentials,
   authHeader: string,
   body: string,
   options: ResilienceOptions = { enableCircuitBreaker: true }
-): Promise<HttpResponse> {
-  const config = wrapXsuaaPostRequestHeader(authHeader, body);
+): Promise<AxiosResponse> {
+  // const config = wrapXsuaaPostRequestHeader(authHeader, body);
   const targetUri =
     typeof tokenServiceUrlOrXsuaaServiceCredentials === 'string'
       ? tokenServiceUrlOrXsuaaServiceCredentials
       : getTokenServiceUrl(tokenServiceUrlOrXsuaaServiceCredentials);
 
-  let destination: Destination = { url: targetUri, proxyType: 'Internet' };
-  if (proxyStrategy(destination) === ProxyStrategy.INTERNET_PROXY) {
-    destination = addProxyConfigurationInternet(destination);
-  }
+  const config: AxiosRequestConfig = {
+    ...urlAndAgent(targetUri),
+    proxy: false,
```

Comment:
in line 279, the proxy value is set to `false`, so we can remove duplicate value.
I have another question below about the value `false`.